lib/firmware.ex

```diff
@@ -132,6 +132,22 @@ defmodule Nerves.Firmware do
   end
 
   @doc """
+  Apply a 1 or 2-phase nerves update
+
+  Applies /tmp/finalize.fw if with `on-reboot` task if exists,
```

Comment:
I feel like there's a grammatical error here, but I can't figure out what the actual intent is. ;)